crates/agglayer-node/src/rpc/mod.rs

```diff
@@ -258,12 +271,26 @@ where
         Ok(status.to_string())
     }
 
-    async fn send_certificate(&self, certificate: Certificate) -> RpcResult<()> {
-        _ = self.pending_store.insert_pending_certificate(
-            certificate.network_id,
-            certificate.height,
-            &certificate,
-        );
+    async fn send_certificate(&self, certificate: Certificate) -> RpcResult<CertificateId> {
+        let id = certificate.hash();
+        // TODO: Batch the different queries.
+        // Insert the certificate header into the state store.
+        _ = self
```

Comment:
Have not seen the `_ = foo();` construct. How does it work? I always used `let _ = foo();`.

Reply:
That's [destructuring-assignment](https://rust-lang.github.io/rfcs/2909-destructuring-assignment.html) RFC, basically it is the same things as `let _` but in a more concise and clear way.